openmc/geometry.py

```diff
@@ -42,7 +42,7 @@ def root_universe(self, root_universe):
 
         self._root_universe = root_universe
 
-    def get_offset(self, path, filter_offset):
+    def get_cell_instance(self, path):
         """Returns the corresponding location in the results array for a given path and
```

Comment:
Although it is separate from the impetus for this PR, do you mind expanding this docstring to say what you think it should say to a novice user?